serving/src/test/java/feast/serving/it/ServingServiceIT.java

```diff
@@ -419,4 +419,75 @@ public void shouldReturnNotFoundForDiffType() {
 
     assertEquals(expectedFieldValuesList, featureResponse.getFieldValuesList());
   }
+
+  @Test
+  public void shouldReturnNotFoundForUpdatedType() {
+    String projectName = "default";
+    String entityName = "driver_id";
+    String featureTableName = "rides";
+
+    ImmutableList<String> entities = ImmutableList.of(entityName);
+    ImmutableMap<String, ValueProto.ValueType.Enum> features =
+        ImmutableMap.of(
+            "trip_cost",
+            ValueProto.ValueType.Enum.INT64,
+            "trip_distance",
+            ValueProto.ValueType.Enum.STRING,
+            "trip_empty",
+            ValueProto.ValueType.Enum.DOUBLE,
+            "trip_wrong_type",
+            ValueProto.ValueType.Enum.STRING);
+
+    TestUtils.applyFeatureTable(
+        coreClient, projectName, featureTableName, entities, features, 7200);
+
+    // Sleep is necessary to ensure caching (every 10s) of updated FeatureTable is done
+    try {
```

Comment:
is it possible to manually trigger cache invalidation?
or at least configure it to every second for example?

Reply:
Unfortunately, we set the number of seconds [here](https://github.com/feast-dev/feast/blob/master/serving/src/main/java/feast/serving/config/SpecServiceConfig.java#L40), I don't think it's possible to configure this setting.

Reply:
but it can be moved to Spring config, right?

Reply:
You mean make it configurable thru serving's application.yml?

Reply:
yep